app/kluby/(catalog)/page.tsx

```diff
@@ -0,0 +1,5 @@
+import { TeamsCatalog } from '@/components/catalogs/teams/teams-catalog'
+
+const PlayersCatalogPage = () => <TeamsCatalog />
+
+export default PlayersCatalogPage
```

Comment:
🌝 + could you rename everything to be CLUB name related instead of TEAMS ? in polish team is a 'zespół'
```suggestion
const ClubsCatalogPage = () => <TeamsCatalog />

export default ClubsCatalogPage
```

Reply:
oh my gooooooood 😠